### PR TITLE
EDM-190/enable unlimited sob access behind feature toggle

### PR DESCRIPTION
**Pull request by jefftmarks**
## Summary

- *This work is behind a feature toggle (flipper):* YES, :sob_updated_design
- Post-911 Statement of Benefits Tool (SOB) relies on chapter33 endpoint of upstream DGI service LTS, which experiences nightly downtime. Operating hours were hard-coded into vets-api to gatekeep requests from flowing upstream during expected downtime. As a result, SOB is not availabel 24/7. DGI has informed our team there has been a deployment to inable 24/7 access to LTS. This PR is first step in testing 24/7 availability of SOB tool.
- Not a bug
-Solution: we've put the hard-coded operating hours behind a feature toggle to test the 24/7 availability. If successful, we will remove the hard-coded operating hours and related logic altogether. In addition, same toggle has been integrated into backend statuses controller, the show action of which uniquely relates to the SOB downtime issue and can probably be removed once 24/7 SOB is confirmed.
- Education Data Migration team, which owns SOB under @department-of-veterans-affairs/govcio-vfep-codereviewers. We don't specifically own backend statuses controller but will work with platform to confirm if in next steps we can remove show  action. 
- Success criteria for flipper is to test all updates to SOB tool. Specifically for this PR, it will test the successful disabling of SOB hard-coded downtime.

## Related issue(s)

- [EDM-190: (SOB) Verify LTS is running 24/7](https://jira.devops.va.gov/browse/EDM-190)

## Testing done

- [x] *New code is covered by unit tests*
- Old behavior: Page for [va.gov/education/gi-bill/post-9-11/ch-33-benefit](https://www.va.gov/education/check-post-9-11-gi-bill-benefits/) will show red warning banner when SOB service is not available during LTS downtime
- To test new changes, access same page in staging or production during old downtime hours (before 6am ET or after 10pm ET) and confirm the warning banner is not there and you can still sign in to access SOB
- Tests have been updated to reflect flipper integration as well as notes for which tests can be removed once SOB is confirmed 24/7
  - QA has already tested existing state of downtime and will test new state once change deployed. After which we will create tickets to remove logic and static references to downtime.

## What areas of the site does it impact?
- Post-911 (chapter 33) Statement of Benefits Tool
- backened statuses controller

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -508,7 +508,7 @@ app/swagger/swagger/requests/messages @department-of-veterans-affairs/vfs-mhv-se
 app/swagger/swagger/requests/mvi_users.rb @department-of-veterans-affairs/octo-identity
 app/swagger/swagger/requests/my_va/submission_statuses.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/swagger/swagger/requests/onsite_notifications.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/swagger/swagger/v1/requests/post911_gi_bill_statuses.rb @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-iir
+app/swagger/swagger/v1/requests/post911_gi_bill_statuses.rb @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/govcio-vfep-codereviewers
```
Comment (RachalCassity) [inline .github/CODEOWNERS:511]: Should all va-iir files be owned by govcio-vfep-codereviewers?

File: spec/controllers/v1/post911_gi_bill_statuses_controller_spec.rb [hunk 2]
```diff
@@ -63,9 +65,14 @@
     end
   end
 
+  # TO-DO: Remove this suite of tests after transition of LTS to 24/7 availability
   context 'outside working hours' do
     # midnight
-    before { Timecop.freeze(tz.parse('2nd Feb 1993 00:00:00')) }
+    before do
+      Flipper.disable(:sob_updated_design)
```
Comment (RachalCassity) [inline spec/controllers/v1/post911_gi_bill_statuses_controller_spec.rb:72]: The current specs do not test the new code when the sob_updated_design is enabled. Can you please make create specs for  Flipper.enable(:sob_updated_design)?

File: spec/requests/v0/backend_status_spec.rb [hunk 3]
```diff
@@ -26,6 +28,8 @@
     end
 
     context 'for the gibs service' do
+      before { Flipper.disable(:sob_updated_design) }
```
Comment (RachalCassity) [inline spec/requests/v0/backend_status_spec.rb:31]: BackendServices::GI_BILL_STATUS was removed in the backend_statuses_controller. Can you please remove traces from this spec?

**Review by RachalCassity — COMMENTED** [review]
Should all files owned by `va-iir `be owned by `govcio-vfep-codereviewers`?

**Comment by jefftmarks** [comment]
> Should all files owned by `va-iir `be owned by `govcio-vfep-codereviewers`?

@RachalCassity we only took over code ownership of the Post-911 Statement of Benefits tool from the IIR team. There could be other files still under their ownership. Also, I saw you left comments about spec file updates but now I can't find them. Would you still like me to implement those changes? Thank you!

**Review by ericboehs — DISMISSED** [review]

**Review by binq — APPROVED** [review]
TGTM

**Comment by jefftmarks** [comment]
@ericboehs I move codeowners file change into a separate PR. Otherwise no other changes. Could you re-review when you have change? Thank you!

**Review by ericboehs — APPROVED** [review]

**Pull request merged**